src/components/MouseFollower/index.tsx

```diff
@@ -1,18 +1,26 @@
 import React from "react";
 
-import styles from "./style.css";
-
```

Comment:
Style sheet was only three lines so I deleted it (it set the `follower` to use absolute positioning)